elixir/apps/api/lib/api/client/channel.ex

```diff
@@ -73,16 +75,28 @@ defmodule API.Client.Channel do
     end
   end
 
+  # Message is scheduled by schedule_expiration/1 on topic join to be sent
+  # when the client token/subject expires
   def handle_info(:token_expired, socket) do
     OpenTelemetry.Ctx.attach(socket.assigns.opentelemetry_ctx)
     OpenTelemetry.Tracer.set_current_span(socket.assigns.opentelemetry_span_ctx)
 
     OpenTelemetry.Tracer.with_span "client.token_expired" do
-      push(socket, "token_expired", %{})
+      push(socket, "disconnect", %{"reason" => "token_expired"})
```

Comment:
@conectado now we can push `disconnect` message with different reasons, you can log or ignore the reason.